packages/sfml/sfml.0.09/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+maintainer: "Florent Monnier <monnier.florent@gmail.com>"
+authors: [
+  "Florent Monnier"
+  "Sylvain Boilard"
+]
+
+homepage: "https://github.com/fccm/ocaml-sfml"
+dev-repo: "git+https://github.com/fccm/ocaml-sfml.git"
+bug-reports: "https://github.com/fccm/ocaml-sfml/issues"
+doc: "http://fccm.github.io/ocaml-sfml/"
+
+license: "zlib/png"
+synopsis: "Bindings to the SFML multimedia library"
+description: """
+SFML provides a simple interface to the various components of your PC, to ease
+the development of games and multimedia applications.
+It is composed of five modules: system, window, graphics, audio and network.
+
+SFML homepage: https://www.sfml-dev.org/
+
+This version of the bindings is known to work with SFML 2.5.1
+"""
+tags: [ "bindings" "graphics" "audio" "multimedia" "gamedev" "opengl" "network" ]
+
+depends: [
+  "ocaml" {> "4.05.1" & < "4.12"}
```

Comment:
Otherwise it still fails with
```
#=== ERROR while compiling sfml.0.09 ==========================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-options-only-afl.1 | pinned(https://github.com/fccm/ocaml-sfml/archive/v0.09.tar.gz)
# path                 ~/.opam/4.14/.opam-switch/build/sfml.0.09
# command              ~/.opam/opam-init/hooks/sandbox.sh build make -C src cxx_all
# exit-code            2
# env-file             ~/.opam/log/sfml-10314-9514b5.env
# output-file          ~/.opam/log/sfml-10314-9514b5.out
### output ###
# make: Entering directory '/home/opam/.opam/4.14/.opam-switch/build/sfml.0.09/src'
# ocamldep SF*.ml SF*.mli oo_*.ml oo_*.mli > ml_depend.mk
# g++ -g -Wall -Werror -Wno-deprecated-declarations -fPIC -c -I/home/opam/.opam/4.14/lib/ocaml -I/usr/include/ ./cxx_stubs/sf_caml_conv.cpp
# g++ -g -Wall -Werror -Wno-deprecated-declarations -fPIC -c -I/home/opam/.opam/4.14/lib/ocaml -I/usr/include/ ./cxx_stubs/sf_conv_vectors.cpp
# g++ -g -Wall -Werror -Wno-deprecated-declarations -fPIC -c -I/home/opam/.opam/4.14/lib/ocaml -I/usr/include/ ./cxx_stubs/SFConfig_stub.cpp
# g++ -g -Wall -Werror -Wno-deprecated-declarations -fPIC -c -I/home/opam/.opam/4.14/lib/ocaml -I/usr/include/ ./cxx_stubs/SFTime_stub.cpp
# In file included from ./cxx_stubs/SFTime_stub.cpp:28:
# ./cxx_stubs/sf_caml_conv.hpp:6: error: "Some_val" redefined [-Werror]
#     6 | #define Some_val(v) Field(v,0)
#       | 
# In file included from ./cxx_stubs/sf_caml_incs.hpp:7,
#                  from ./cxx_stubs/SFTime_stub.cpp:27:
# /home/opam/.opam/4.14/lib/ocaml/caml/mlvalues.h:396: note: this is the location of the previous definition
#   396 | #define Some_val(v) Field(v, 0)
#       | 
# cc1plus: all warnings being treated as errors
# make: *** [Makefile:181: SFTime_stub.o] Error 1
# make: Leaving directory '/home/opam/.opam/4.14/.opam-switch/build/sfml.0.09/src'
```
on 4.12.0